meson.build

```diff
@@ -369,7 +369,8 @@ foreach item : [
     ['system', '#include <stdlib.h>', []],
     ['fork', '#include <unistd.h>', []],
     ['nice', '#include <unistd.h>', []],
-    ['pipe2', '#define _GNU_SOURCE\n#include <fcntl.h>\n#include <unistd.h>', []]
+    ['pipe2', '#define _GNU_SOURCE\n#include <fcntl.h>\n#include <unistd.h>', []],
+    ['strlcpy', '#include <string.h>', []],
```

Comment:
I recommend to swap the line with `pipe2` so the longest line will always be on the bottom.